mitxgraders/comparers/comparers.py

```diff
@@ -240,10 +241,28 @@ def vector_span_comparer(comparer_params_evals, student_eval, utils):
     The vector 2*v0 + 3i*v1 = [2, 2+3i, 6i] is in the span of v0 and v1:
     >>> grader(None, '[2, 2 + 3*i, 6*i]')['ok']
     True
+
+    The compaer_params should be list of equal-length vectors:
```

Comment:
Typo in `comparer_params`

Reply:
So, the `r` button on my keyboard only works 30% of the time :) Fixing that soon I hope...

Reply:
Man, that would dive me cazy!